cmake/compiler/gcc/target.cmake

```diff
@@ -45,6 +45,10 @@ foreach(file_name include/stddef.h include-fixed/limits.h)
     OUTPUT_VARIABLE _OUTPUT
     )
   get_filename_component(_OUTPUT "${_OUTPUT}" DIRECTORY)
+  # get absolute pathname
+  cmake_path(NORMAL_PATH _OUTPUT OUTPUT_VARIABLE _OUTPUT)
+  # solve gcc's MSYS pathname, harmless on other hosts
+  string(REGEX REPLACE "^(.):(.)" "\\2\\1\\2" _OUTPUT "${_OUTPUT}")
```

Comment:
should define a general function to handle this, since there is likely going to be more places that require this (now and in the future as well).